css/style.css

```diff
@@ -79,11 +78,9 @@ header {
   display: flex;
   flex-direction: row;
   justify-content: space-around;
-  align-items: center;
 }
 
 h1 {
-  margin-left: 165px;
   font-size: 40px;
   font-weight: 700;
   letter-spacing: 0.2px;
```

Comment:
Please add properties with better specificity, as this will override all h1 properties.
Example:
```
  .className h1 {
   .......
  }
```